webpack.test.config.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+var webpack = require('webpack'),
+    config = require('./webpack.common.config.js');
+
+// Only use the test entry file
+config.entry = {
+    'js/testRunner': __dirname + '/assets/tests/testRunner.js'
+};
+// Put a dummy value in place of the django reverse module in test mode
+config.externals = {
+    reverse: "undefined"
```

Comment:
Yes. The values in the `externals` hash are the _names_ of the global variables that are returned when requesting that module.  So in this case, when a module requires `reverse` in the test bundle, it will get the global variable _named_ `undefined`.